src/workloads/execution/TimeSeriesSortCompound.yml

```diff
@@ -0,0 +1,117 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This test exercises the behavior of the time series bounded sorter with a compound sort.
```

Comment:
It might be helpful to add a comment about how the data looks, or mention that its the same as the description in TimeSeriesSortOverlappingBuckets.yml